packages/api-explorer/src/scenes/MethodScene/MethodScene.tsx

```diff
@@ -72,10 +74,10 @@ export const MethodScene: FC<DocMethodProps> = ({ api }) => {
       <DocMarkdown source={method.description} specKey={specKey} />
       <DocSDKs api={api} method={method} />
       {seeTypes.length > 0 && (
-        <>
-          Referenced types:
+        <Flex mb="large">
+          <Flex mr="xsmall">Referenced types:</Flex>
           <DocReferences items={seeTypes} api={api} specKey={specKey} />
-        </>
+        </Flex>
```

Comment:
Can wrap this in a `<Space>` instead of `Flex`

I don't think you need the interior `Flex` a `span` or `div` should work

Reply:
Good catch, updated.